src/ui/components/Rating/index.js

```diff
@@ -66,18 +71,35 @@ export default class Rating extends React.Component {
   }
 
   render() {
-    const { readOnly, size } = this.props;
+    const { i18n, rating, readOnly, size } = this.props;
     const cls = classNames('Rating', {
       'Rating--editable': !readOnly,
       'Rating--small': size === 'small',
```

Comment:
Maybe not super-related to this PR but is there a set of valid sizes? I feel like we should throw if the size isn't valid, especially as the values aren't passed around as constants.